src/openms/source/ANALYSIS/ID/AccurateMassSearchEngine.cpp

```diff
@@ -960,11 +960,11 @@ namespace OpenMS
 
     bool isotope_export = param_.getValue("mzTab:exportIsotopeIntensities").toString() == "true";
 
-    for (QueryResultsTable::const_iterator tab_it = overall_results.begin(); tab_it != overall_results.end(); ++tab_it)
+    for (const auto& result : overall_results)
     {
-      for (Size hit_idx = 0; hit_idx < tab_it->size(); ++hit_idx)
+      for (Size hit_idx = 0; hit_idx < result.size(); ++hit_idx)
       {
-        std::vector<String> matching_ids = (*tab_it)[hit_idx].getMatchingHMDBids();
+        std::vector<String> matching_ids = (result)[hit_idx].getMatchingHMDBids();
```

Comment:
Please remove brackets that are not needed any more after the conversion
```suggestion
        std::vector<String> matching_ids = result[hit_idx].getMatchingHMDBids();
```

Reply:
also below a few times